src/r_draw.c

```diff
@@ -47,8 +44,8 @@ int  scaledviewheight;        // killough 11/98
 int  viewheight;
 int  viewwindowx;
 int  viewwindowy; 
-byte *ylookup[MAXHEIGHT]; 
-int  columnofs[MAXWIDTH]; 
+byte **ylookup; // [Nugget] Dynamic arrays
```

Comment:
Global variables should always be implicitly initialized to `NULL`, but I would explicitly add it here anyway.